src/components/admin/ScraperDashboard.tsx

```
@@ -0,0 +1,230 @@
+            <select 
+              value={mode} 
+              onChange={(e) => setMode(e.target.value as any)}
+              className="w-full p-2 border rounded dark:bg-gray-700"
+            >
+              <option value="NEW">New Product Scan (Default)</option>
+              <option value="BACKFILL">Backfill (Resume History)</option>
+            </select>
+          </div>
+          
+          <div>
+             <label className="block text-sm font-medium mb-1">Page Limit</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)